tokens.xml

```diff
@@ -8530,6 +8659,52 @@ Sacrifice this creature: This creature deals 1 damage to target creature or play
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Zombie    </name>
+            <text>Decayed</text>
+            <prop>
+                <colors>B</colors>
+                <type>Token Creature — Zombie</type>
+                <maintype>Creature</maintype>
+                <cmc>0</cmc>
+                <pt>2/2</pt>
+            </prop>
+            <set picURL="https://media.wizards.com/2021/mid/en_gQ1kYIpAvu.png">MID</set>
+            <reverse-related count="x">Crowded Crypt</reverse-related>
+            <reverse-related>Curse of the Restless Dead</reverse-related>
+            <reverse-related count="x" exclude="exclude">Curse of the Restless Dead</reverse-related>
+            <reverse-related count="2">Diregraf Horde</reverse-related>
+            <reverse-related>Falcon Abomination</reverse-related>
+            <reverse-related>Flip the Switch</reverse-related>
+            <reverse-related count="x">Ghoulcaller's Harvest</reverse-related>
+            <reverse-related>Ghoulish Procession</reverse-related>
+            <reverse-related>Hobbling Zombie</reverse-related>
+            <reverse-related>Jadar, Ghoulcaller of Nephalia</reverse-related>
+            <reverse-related>No Way Out</reverse-related>
+            <reverse-related>Poppet Stitcher</reverse-related>
+            <reverse-related>Revenge of the Drowned</reverse-related>
+            <reverse-related>Rotten Reunion</reverse-related>
+            <reverse-related>Startle</reverse-related>
+            <reverse-related count="x">Tainted Adversary</reverse-related>
```

Comment:
This card can only create even numbers of tokens, so the minimum here should be 2, not 1.